crates/fmt/testdata/Yul/fmt.sol

```diff
@@ -142,9 +148,18 @@ contract Yul {
         assembly "evmasm" ("memory-safe") {}
 
         assembly {
-            for { let i := 0 } lt(i, 10) { i := add(i, 1) } { mstore(i, 7) }
+            for {
+                let i := 0
+            } lt(i, 10) {
+                i := add(i, 1)
+            } {
+                mstore(i, 7)
+            }
 
-            function sample(x, y) ->
+            function sample(
```

Comment:
this is probably wrong? the params can fit in one line and we should split only the returns